ddl/ddl_api.go

```diff
@@ -1759,18 +1768,30 @@ func resolveDefaultTableCharsetAndCollation(tbInfo *model.TableInfo, dbCharset s
 	return
 }
 
-func findTableOptionCharset(options []*ast.TableOption) string {
-	var tableCharset string
+func findTableOptionCharsetAndCollate(options []*ast.TableOption) (tableCharset, tableCollate string) {
```

Comment:
Please use this function instead:

https://github.com/pingcap/tidb/blob/aaaf2546dd0bd0636e125741a90d1ffb15222292/ddl/ddl_api.go#L1841

I think we can remove `findTableOptionCharsetAndCollate`